Project Walking/Source code/app/src/main/java/com/teamapricot/projectwalking/controller/NavigationController.java

```diff
@@ -47,6 +48,11 @@ public NavigationController(NavigationModel navigationModel, AppCompatActivity a
         this.activity = activity;
         roadManager = new OSRMRoadManager(activity, "Fun Walking");
         ((OSRMRoadManager) roadManager).setMean(OSRMRoadManager.MEAN_BY_FOOT);
+        Database.getDatabase(activity.getApplicationContext()).thenAccept(database -> {
+            routeDao = database.routeDao();
+            // navigationModel.initDestination(roadManager);
+            routeDao.deleteOpenRoutes();
+        });
```

Comment:
Commented code! 😄

Reply:
Added a bit of explanation to it too. Easy to forget to do it if it isn't mentioned, I think.